cove_iati/lib/iati.py

```diff
@@ -447,3 +487,118 @@ def organisation_identifier_count(tree):
     unique_identifiers = list(set(identifiers))
 
     return len(unique_identifiers)
+
+
+ACTIVITY_PREFIX = '/iati-activities/iati-activity'
+
+
+def check_activity_org_refs(tree):
+    root = tree.getroot()
+
+    publisher_request = requests.get("https://codelists.codeforiati.org/api/json/en/ReportingOrganisation.json")
+    registration_agency_request = requests.get("https://codelists.codeforiati.org/api/json/en/OrganisationRegistrationAgency.json")
+
+    if publisher_request.status_code != 200 or registration_agency_request.status_code != 200:
+        return {"error": "Unable to fetch data to do organisation checks", 'not_found_orgs_count': 0}
```

Comment:
This doesn't catch ConnectionError or SSLError (which I tried out by editing my hosts file).